zboxcore/sdk/attributesworker.go

```diff
@@ -43,63 +43,39 @@ func (ar *AttributesRequest) getObjectTreeFromBlobber(
 func (ar *AttributesRequest) updateBlobberObjectAttributes(
 	blobber *blockchain.StorageNode, blobberIdx int) (
 	re fileref.RefEntity, err error) {
+	var s strings.Builder
 
 	re, err = ar.getObjectTreeFromBlobber(ar.blobbers[blobberIdx])
 	if err != nil {
 		return
 	}
 
-	var (
-		body bytes.Buffer
-		form = multipart.NewWriter(&body)
-	)
-
-	form.WriteField("connection_id", ar.connectionID)
-	form.WriteField("path", ar.remotefilepath)
-	form.WriteField("attributes", ar.attributes)
-
-	form.Close()
+	remoteFilePathHash := fileref.GetReferenceLookup(ar.allocationID, ar.remotefilepath)
 
-	var httpreq *http.Request
-	httpreq, err = zboxutil.NewAttributesRequest(blobber.Baseurl,
-		ar.allocationTx, &body)
+	respRaw, err := blobberClient.UpdateObjectAttributes(blobber.Baseurl, &blobbergrpc.UpdateObjectAttributesRequest{
+		Path:         ar.remotefilepath,
+		PathHash:     remoteFilePathHash,
+		Allocation:   ar.allocationTx,
+		ConnectionId: ar.connectionID,
+		Attributes:   ar.attributes,
+	})
 	if err != nil {
-		Logger.Error(blobber.Baseurl,
-			"Error creating update attributes request", err)
+		Logger.Error("could not update object attributes from blobber -" + blobber.Baseurl + " - " + err.Error())
 		return
```

Comment:
Can we wrap & return error instead of nil returns

Reply:
it's does return the error still,but i can wrap them.